echo-scheduler/src/main/groovy/com/netflix/spinnaker/echo/scheduler/actions/pipeline/impl/PipelineTriggerConverter.groovy

```diff
@@ -56,7 +56,7 @@ class PipelineTriggerConverter {
       triggerBuilder.runAsUser(parameters.runAsUser)
     }
 
-    def existingPipeline = pipelineCache.getPipelines().toBlocking().first().find { it.id == parameters.id }
+    def existingPipeline = pipelineCache.getPipelines().find { it.id == parameters.id }
```

Comment:
Given that this blocked before, should we be using `getPipelinesSync` here?  (This looks like it will throw an NPE if called before the first polling cycle.)